packages/studio/src/store/models/slices/PageSlice.ts

```diff
@@ -68,7 +68,9 @@ interface PageSliceActions {
   setModuleUUIDBeingEdited: (moduleStateUUID: string | undefined) => void;
 
   setActiveEntityFile: (activeEntityFile?: string) => void;
-  updateActivePageEntities: (parentFolder: string) => Promise<void>;
+  updateActivePageEntities: (
```

Comment:
Could we call this `setActivePageEntities`? By moving the import logic out, we've effectively just made this method a setter. I think the `update` verbiage implies some extra work before the `set` is called.